pages/discover.js

```diff
@@ -113,11 +112,11 @@ export default function Discover() {
             Protecting the data itself from end-to-end.
           </h3>
           <div className="pt-4">
-            CDS secures individuals and companies privacy by design. When data is shared, CDS
-            ensures that only the recipient can decrypt and read it on their device. Moreover,
-            before being sent, the data is structured in an object that cannot be modified without
-            leaving a trace. Thus, the emitter’s data is secured and is totally trustworthy for the
-            recipient.
+            The CDS protects the privacy of individuals and companies by design. When data is
+            shared, CDS ensures that only the recipient can decrypt and read it on their device.
+            Furthermore, before being sent, the data is structured in an object that cannot be
```

Comment:
let's simplify all the blocks here, and not talk about blocks etcs, its more the purpose of the how it works section